src/Core/System/Snippet/Struct/InvalidPluralizationCollection.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\System\Snippet\Struct;
+
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Struct\Collection;
+
+/**
+ * @extends Collection<InvalidPluralizationStruct>
+ */
+#[Package('discovery')]
+class InvalidPluralizationCollection extends Collection
```

Comment:
to have some additional type safety, you can implement the `getExpectedClass` with returning `InvalidPluralizationStruct` class name